staging/src/k8s.io/component-base/metrics/processstarttime.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs"
+	"k8s.io/klog"
+	"os"
+	"time"
+)
+
+var processStartTime = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Name: "process_start_time_seconds",
+		Help: "Start time of the process since unix epoch in seconds.",
```

Comment:
Why is this not in a human readable format - example one of the formats supports by go - https://golang.org/pkg/time/#Time.Format?

Reply:
Oh, this has to be conformant to the prom go_lang client's implementation of this metric. We opted not to use their implementation because it drags in many other metrics.

Reply:
For ref: https://github.com/prometheus/client_golang/issues/578

Reply:
Please explain that in a comment for future readers of the code?